lib/elixir/test/elixir/kernel/alias_test.exs

```diff
@@ -44,6 +44,23 @@ defmodule Kernel.AliasTest do
   test "nested elixir alias" do
     assert Kernel.AliasTest.Elixir.sample == 1
   end
+
+  defmodule Multi.SampleA do
+    def main(), do: "A"
+  end
+
+  defmodule Multi.SampleB do
+    def main(), do: "B"
+  end
+
+  test "multi-call" do
+    alias Multi.{
```

Comment:
Let's add a test for `alias Multi.{}` too. It could be here or in the imports one (the imports one may be easier to test in this particular case where we have a no-op).